core/src/main/scala/org/ensime/core/javac/JavaCompiler.scala

```diff
@@ -46,7 +48,8 @@ class JavaCompiler(
   private val workingSet = new ConcurrentHashMap[String, JavaFileObject]()
 
   private implicit def charset: Charset = Charset.defaultCharset() // how can we infer this?
-
+  val context = new com.sun.tools.javac.util.Context() //Need to have a single context and not one for each source??
```

Comment:
this looks like an explanation for the line, rather than a suggestion to improve it (which is confusing). Best create a ticket and reference it.